director/director_ui.go

```diff
@@ -0,0 +1,75 @@
+package director
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+)
+
+type (
+	listServerRequest struct {
+		ServerType string `form:"server_type,omitempty"` // "cache" or "origin"
```

Comment:
What is this omit empty doing?

Reply:
Thanks for pointing out. It should be removed it as this is a request struct not response